grapher/scan.py

```diff
@@ -132,11 +132,22 @@ def pkgToUnit(pkg: Dict) -> List[Unit]:
     if len(test_files) == 0:
         return [unit]
 
+    # Collect files as candidates of modules.
+    modules = []
```

Comment:
Got it, makes sense. Can you factor this out into a separate function for readability and then add a test case repository (or modify one of the existing test case repositories to cover this)?